.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v4
+
+ #   - name: Install necessary packages
+ #     uses: awalsh128/cache-apt-pkgs-action@v1
+ #     with:
+ #       packages: libgmp-dev libboost-dev libboost-filesystem-dev libboost-system-dev libreadline-dev gcc g++ make
+ #       execute_install_scripts: true
+ #       version: 1.0
```

Comment:
This is intentionally commented out, but left in here.
If the underlying action is fixed at some point, this will speedup future CI runs.